packages/react-components/react-positioning/src/types.ts

```diff
@@ -62,10 +62,10 @@ export interface PositioningOptions {
   align?: Alignment;
 
   /** The element which will define the boundaries of the positioned element for the flip behavior. */
-  flipBoundary?: Boundary;
+  flipBoundary?: Boundary | null;
```

Comment:
these options should accept `null` because that's how React types an unset DOM ref